tests/Unit/Type/ItemPriceTest.php

```diff
@@ -836,12 +925,61 @@ public function taxesProvider()
             [
                 false,
                 [
-                    [new TaxPrice(10, 'exclusive'), new TaxPrice(15, 'exclusive')],
-                    [new TaxPrice(25, 'exclusive')]
+                    [new TaxPrice(10, TaxPrice::EXCLUSIVE), new TaxPrice(15, TaxPrice::EXCLUSIVE)],
+                    [new TaxPrice(25, TaxPrice::EXCLUSIVE)]
                 ],
                 2,
                 3
             ]
         ];
     }
+
+    /**
+     * @covers ::resetTaxes
+     * @uses Blesta\Pricing\Type\ItemPrice::__construct
+     * @uses Blesta\Pricing\Type\ItemPrice::resetDiscountSubtotal
+     * @uses Blesta\Pricing\Type\ItemPrice::excludeTax
+     * @uses Blesta\Pricing\Type\ItemPrice::subtotal
+     * @uses Blesta\Pricing\Type\UnitPrice::__construct
+     * @uses Blesta\Pricing\Type\UnitPrice::setPrice
+     * @uses Blesta\Pricing\Type\UnitPrice::setQty
+     * @uses Blesta\Pricing\Type\UnitPrice::setKey
+     * @uses Blesta\Pricing\Type\UnitPrice::total
+     * @uses Blesta\Pricing\Modifier\AbstractPriceModifier::type
+     */
+    public function testResetTaxes()
+    {
+        $item = new ItemPrice(10);
+        $temp_item = new ItemPrice(10);
+
+        $item->excludeTax(TaxPrice::EXCLUSIVE);
+        $item->resetTaxes();
+        $this->assertEquals($item, $temp_item);
```

Comment:
I think a better test here would be to verify the state change of the property via "assertAttributeEquals", e.g.

```
$item = new ItemPrice(10);
$default_tax_types = [TaxPrice::INCLUSIVE => false, TaxPrice::EXCLUSIVE => true];
$this->assertAttributeEquals($default_tax_types, 'tax_types', $item);

$item->excludeTax(TaxPrice::EXCLUSIVE);
$this->assertAttributeEquals([TaxPrice::INCLUSIVE => false, TaxPrice::EXCLUSIVE => false]);

$item->resetTaxes();
$this->assertAttributeEquals($default_tax_types, 'tax_types', $item);
```